zproject/settings.py

```diff
@@ -157,6 +157,10 @@ def get_secret(key):
     'ENABLE_FEEDBACK': PRODUCTION,
     'FEEDBACK_EMAIL': None,
 
+    # Max state storage per user
+    # TODO: Add this to zproject/prod_settings_template.py once stateful bots are fully functional.
+    'USER_STATE_SIZE_LIMIT': 10000000,
```

Comment:
Maybe we should name this "BOT_..."? I initially chose user because the state odel technically allows for any user to store state, but BOT_ would probably make more sense to an administrator.

Reply:
Since this isn't an exported API, we can plan to change it later.